packages/cli/src/commands/destroy/scaffold/scaffold.js

```diff
@@ -13,6 +21,54 @@ export const command = 'scaffold <model>'
 export const description =
   'Destroy pages, SDL, and Services files based on a given DB schema Model'
 
+const removeRoutesWithSet = async (model, path) => {
+  const routes = await scaffoldRoutes({ model, path })
+  const routeNames = routes.map(extractRouteName)
```

Comment:
naming the function passed to map—super readable, nice 👍

Reply:
Oh I just took reference from `generate` one. Credit goes to them! 👍